mycore-solr/src/main/java/org/mycore/solr/proxy/MCRSolrProxyServlet.java

```diff
@@ -275,6 +276,14 @@ private void handleQuery(String queryHandlerPath, HttpServletRequest request, Ht
         solrHttpMethod.releaseConnection();
     }
 
+    private void filterParams(ModifiableSolrParams solrParameter) {
```

Comment:
I think this is betterreadable:
```
MCRConfiguration2.getString("MCR.Solr.Disallowed.Facets")
            .map(MCRConfiguration2::splitValue).orElse(Stream.empty())
            .forEach(disallowedFacet -> {
                solrParameter.remove("facet.field", disallowedFacet);
            });
```

Reply:
To be honest I think this just a matter of taste.